README.md

```diff
@@ -1,2 +1,13 @@
 # swc-conda-recipes
 Conda meta-packages for helping with workshop installation
+
+Core concepts:
+  * environment.yml should be downloaded by end-users into a folder by itself, followed by
+
+```
+conda-env create 
+```
+
+in that folder.  This creates an env named like the folder containing environment.yml.  Please keep the "name" field in environment.yml consistent with the folder name.
+
+  * environment.yml files do not support specifying different packages based on platform (selectors).  Conda recipes do.  We create metapackages that behave differently on different platforms, and have the environment.yml install those packages.  To change the environment people get, change these metapackages (currently here, as swc-[topic])
```

Comment:
I've been up to my chin in conda recipes lately, and wrote the environment.yml file first with that syntax, trying to install git on *nix, and git_windows on windows.  When that didn't work at all, and I couldn't find anything on Google explaining why, I went to this approach.  I haven't confirmed this with an in-depth explanation from conda-env source code, but I am pretty sure it's true.